tests/js/spec/stores/organizationStore.spec.jsx

```diff
@@ -68,4 +70,41 @@ describe('OrganizationStore', function() {
       dirty: false,
     });
   });
+
+  it('loads in sorted teams', async function() {
+    const organization = TestStubs.Organization();
+    OrganizationActions.update(organization);
+    // wait for action to get dispatched to store
+    await tick();
+
+    const teamA = TestStubs.Team({slug: 'a_team'});
+    const teamB = TestStubs.Team({slug: 'b_team'});
+    const teams = [teamB, teamA];
+    TeamActions.loadTeams(teams);
+    // wait for action to get dispatched to store
+    await tick();
+
+    // verify existence and sorted order of loaded teams
+    expect(OrganizationStore.get().organization.teams).toEqual([teamA, teamB]);
+  });
+
+  it('loads in sorted projects', async function() {
+    const organization = TestStubs.Organization();
+    OrganizationActions.update(organization);
+    // wait for action to get dispatched to store
+    await tick();
+
+    const mockProjectA = TestStubs.Project({slug: 'a'});
+    const mockProjectB = TestStubs.Project({slug: 'b'});
```

Comment:
Nit: Variable names should be consistent in `loads in sorted teams` and `loads in sorted projects`.

Reply:
ooh tru tru good call